pkg/controller/rollout/rollout_progressing.go

```diff
@@ -149,10 +165,15 @@ func (r *RolloutReconciler) doProgressingInitializing(c *util.RolloutContext) (b
 		klog.Infof("verify rollout(%s/%s) TrafficRouting, and wait a moment", c.Rollout.Namespace, c.Rollout.Name)
 		return false, nil
 	}
+
+	// handler TrafficRouting
+	if c.Rollout.Annotations[v1alpha1.TrafficRoutingAnnotation] != "" {
+		return r.handlerTrafficRouting(c.Rollout.Namespace, c.Rollout.Name, c.Rollout.Annotations[v1alpha1.TrafficRoutingAnnotation])
```

Comment:
plz add ut in rollout_progressing_test.go to cover rollout with TrafficRoutingAnnotation